docs/visual-basic/language-reference/error-messages/clipboard-format-is-not-valid.md

```diff
@@ -12,7 +12,7 @@ The specified Clipboard format is incompatible with the method being executed. A
   
 -   Using the Clipboard's `GetData` or `SetData` method with a Clipboard format other than `vbCFBitmap`, `vbCFDIB`, or `vbCFMetafile`.  
   
--   Using the `DataObject``GetData` method or `SetData` method with a Clipboard format in the range reserved by Microsoft Windows for registered formats (&HC000-&HFFFF), when that Clipboard format has not been registered with Microsoft Windows.  
+-   Using the `DataObject` `GetData` method or `SetData` method with a Clipboard format in the range reserved by Microsoft Windows for registered formats (&HC000-&HFFFF), when that Clipboard format has not been registered with Microsoft Windows.  
```

Comment:
nit unrelated to your PR: this is awkward. Perhaps, "Using the \`GetData\` or \`SetData\` methods of a `\DataObject\`"?